eodhp_utils/runner.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from pulsar import Client, ConsumerDeadLetterPolicy, ConsumerType
+
+
+def run(messagers_dict: dict, subscription_name: str):
+    """Run loop to monitor arrival of pulsar messages on a given topic.
+
+    Example usage:
+    annotations_messager = AnnotationsMessager(s3_client=s3_client, output_bucket=destination_bucket)
+    run(
+        {
+            "transformed-annotations": annotations_messager
+        },
+        "annotations-ingester",
+    )
+    """
+
+    pulsar_url = os.environ.get("PULSAR_URL")
+    client = Client(pulsar_url)
```

Comment:
Transformers written as messager will need the Pulsar client as well because the Producer used for sending messages is an argument to the Messager class. Perhaps we should add a `get_pulsar_client()` method in this file which creates the client on the first call and always returns the same one on later calls? Then it could be called both from here and from individual harvest pipeline startup code than needs access to it.